### PR TITLE
Add Ambient Alpha Banner to Homepage

### PR DESCRIPTION
**Pull request by mike-weiner**
Fixes: #13478 

This PR adds a banner to the homepage that brings attention to the Ambient Alpha release apart of Istio v1.18. A screenshot below demonstrates what this banner will look like.

Screenshot 2023-07-28 at 3 44 36 PM

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by craigbox** [comment]
I suggest we wait until the current banners are done.

TBH, I also suggest we don't quite heavily promote something that is only in alpha! Do we need the full attention of the community here - is there a specific CTA (testing?) we want at this stage?

**Comment by mike-weiner** [comment]
We can definitely hold off. The issue (#13478) was opened about a month back - so I figured this was on the "want" list for sometime soon-ish. A decision could have been made to hold off on this and it was just never added to the original issue. Will have to get input from the original issue creator(s) on that.

**Comment by rootsongjc** [comment]
> I suggest we wait until the current banners are done.
> 
> TBH, I also suggest we don't quite heavily promote something that is only in alpha! Do we need the full attention of the community here - is there a specific CTA (testing?) we want at this stage?

I think that having two banners on the website is too much. It's distracting and makes the website look cluttered. I think it would be better to just have one banner.

**Comment by ericvn** [comment]
@craigbox Does it make sense to try and do this now or would it dilute the 1.19 release if we tried to release it then (and we have the multiple banner issue you mention above)?

**Comment by craigbox** [comment]
Promote ambient to beta. 🙂

**Comment by mike-weiner** [comment]
@craigbox - You're saying to hold off on this until Ambient is in beta?

**Review by craigbox — COMMENTED** [review]
I'm still inclined to leave this out until we have hit Beta, or we have a specific request we want people to follow through on.  Hopefully that's in 1.20 or 1.21?

**Comment by linsun** [comment]
I'm inclined to have this to get more visibility of ambient + its alpha status. we have heard a lot of people asking it.

Once we get beta, we can flip... thoughts? @istio/steering-committee

**Comment by craigbox** [comment]
Yes, we intend to completely revamp the web page when Ambient is in Beta!

**Comment by craigbox** [comment]
Given that, I am going to close this until beta.

**Comment by rvennam** [comment]
I'm with Lin - I think we should direct more users to try out the alpha and find any issues before beta :)

**Comment by craigbox** [comment]
Oh, sorry, I misread what Lin said and thought she meant, only have it at Beta

File: content/en/events/banners/ambient-release.md
Comment (rvennam) [inline content/en/events/banners/ambient-release.md:9]:
```suggestion
{{< gloss "ambient" >}}Ambient{{< /gloss >}} is part of Istio {{< istio_version >}} as alpha status, [try it out!](docs/ops/ambient/getting-started/)
```

Comment (dhawton) [reply content/en/events/banners/ambient-release.md:9]: The entire banner is a link.

Comment (rvennam) [reply content/en/events/banners/ambient-release.md:9]: Ah!

Comment (dhawton) [reply content/en/events/banners/ambient-release.md:9]: Was thinking if the whole banner should link to the getting started page, or the alpha announcement.

File: content/en/events/banners/ambient-release.md
Comment (dhawton) [inline content/en/events/banners/ambient-release.md:6]:
```suggestion
link: https://istio.io/docs/ops/ambient/getting-started/
```

Think we should link to the getting started doc instead since the banner is a recommendation to try it out and we specify a minor version in the banner to indicate what minor version they should try.

Comment (craigbox) [reply content/en/events/banners/ambient-release.md:6]: Agreed. I think I can merge a suggestion as a codeowner.

Comment (mike-weiner) [reply content/en/events/banners/ambient-release.md:6]: @craigbox - Do you want me to go ahead and commit that suggestion?

**Comment by craigbox** [comment]
Please!

**Comment by mike-weiner** [comment]
Suggestion has been committed & builds have started.

**Review by dhawton — APPROVED** [review]

**Comment by linsun** [comment]
Thank you @mike-weiner! apologizes for our slow review - been wanted to have this since last April!!

**Pull request merged**
